src/v3/src/transformer/layout/email/transformEmailAuthenticatorEnroll.ts

```diff
@@ -70,12 +68,12 @@ export const transformEmailAuthenticatorEnroll: IdxStepTransformer = ({ transact
   if (typeof useEmailMagicLink === 'undefined') {
     subTitleElement.options.content = loc('oie.email.enroll.subtitle', 'login');
   } else {
-    const redactedEmailAddress = nextStep.relatesTo?.value?.profile?.email;
```

Comment:
In enrollment flow (which is what this transformer is for) the identifier is pulled from the context instead of the authenticator block of the remediation. My assumption is because for enrolling, we are just using the email that is already displayed at the top of the page (not totally sure on this but just my guess). However, I confirmed this is how v2 does it for the enroll email flow and the test that was enabled verified as such.

Reply:
Interesting, so the identifier in the transaction context is already redacted as well? Or it's not redacted but meant to not be redacted for enrollment?

Reply:
Based on v2, it looks like its not redacted for enrollment, but it is redacted for verification flow. The redacted email is not present in the idx response for enrollment flow.